.stoat/config.yaml

```diff
@@ -21,3 +21,9 @@ plugins:
       metadata:
         name: Documentation
       path: docs/build
+  image_diff:
+    compare_logo_image:
+      metadata:
+        name: Logo
+      path: docs/static/img/logo-128.png
+      baseline: https://stoat-dev--static.stoat.page/branding/128.png
```

Comment:
Should the baseline default to the statically hosted version on the default branch (if it exists)?

Reply:
Yes, that's the plan, but has not been implemented. Currently nothing exists on the default branch. So I have to use a static file for demo and testing.

Reply:
https://github.com/stoat-dev/stoat-action/issues/114